cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -223,6 +260,9 @@ public CloudantClient build() {
         //Build properties and couchdb client
         CouchDbProperties props = new CouchDbProperties(url);
 
+        props.addRequestInterceptors(USER_AGENT_INTERCEPTOR);
```

Comment:
we should just be able to new the interceptor up here and get rid of the static.